desktop/index.js

```diff
@@ -0,0 +1,69 @@
+const app = module.exports = require('express')()
```

Comment:
Like I mentioned in the description, these root express apps for desktop and mobile sides are now where all of their sub-apps are mounted.